sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/PatchSpec.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.models.CosmosPatchItemRequestOptions;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.util.Beta;
+import com.azure.cosmos.CosmosPatchOperations;
+
+import java.util.ArrayList;
+
+import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkNotNull;
+
+@Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public final class PatchSpec {
```

Comment:
`PatchSpec` is in the `com.azure.cosmos.implementation` and hence is not part of the public API. 
we don't expect end user to use this class right?

if so drop all the `Beta` annotations on this class and its methods. Only the beta public apis should be annotated not the implementation.